cumulusci/cli/cci.py

```diff
@@ -178,6 +179,8 @@ def new_func(*args, **kw):
     """Run this command for more information about debugging errors: cci error --help"""
 )
 
+UsageErrors = (CumulusCIUsageError, click.UsageError)
```

Comment:
```suggestion
USAGE_ERRORS = (CumulusCIUsageError, click.UsageError)
```

to match our convention and make sure it doesn't look like it's a class